app/views/spree/admin/invoices/_invoices_table.html.haml

```diff
@@ -0,0 +1,24 @@
+%table.index{"data-hook" => "invoices"}
+  %thead{"data-hook" => "invoice_head"}
+    %tr
+      %th= "#{t('spree.date')}/#{t('spree.time')}"
+      %th= t(:invoice_number)
+      %th= t(:amount)
+      %th= t(:status)
+      %th= t(:file)
+  %tbody
+    - @order.invoices.each do |invoice|
+      - tr_class = cycle('odd', 'even')
```

Comment:
Oh dear, this is really old code. CSS is much better at the odd-even style. But that doesn't need solving in this PR.